api/USERS_GUIDE.md

```diff
@@ -125,6 +125,16 @@ curl -i -X GET "https://www.pgdp.org/api/v1/projects?title=Monster" \
     -H "X-API-KEY: $API_KEY"
 ```
 
+To limit which project fields are included, use the `fields` parameter to pass
+in a comma-separated list of fields to return.
+
+To return only the `projectid`, `title`, and `author` fields:
+```bash
+curl -i -X GET "https://www.pgdp.org/api/v1/projects?fields=projectid,title,author" \
```

Comment:
Based on the last MR about multi fields, should we instead do field[]=title&field[]=author etc.?

Reply:
I get where you're going with that, but I think that will be confusing with this endpoint in particular because we aren't `OR`ing the fields together in any way and they aren't part of the search criteria, for instance:
```
genre[]=Science&genre[]=Sports&field[]=title&field[]=author
```
Thoughts?

Reply:
I see where you are coming from. I think it would make sense to a user of the API if we just explained that all fields that accept multiple values are sent with field[] syntax. Then we need to clarify that multivalued search fields are ORed while distinct values that are relevant to search are ANDed. I think that is less confusing then figuring out how to send multi values depending on how they are used by the API.

In reality though, I've seen it done both ways and am open to either.

Reply:
That makes a lot of sense. The comma approach is nice but we can only use it for strings that might not contain commas (like a field name). I agree it makes more sense to be consistent within the API and use `[]` for multiple values. Let me get the branch changed to use that instead. Thanks for bringing it up and talking through it.